v1/ast/parser_test.go

```diff
@@ -6392,7 +6387,7 @@ func TestMaxParsingRecursionDepth(t *testing.T) {
 	}{
 		{
 			name:        "deeply nested array exceeds default limit",
-			input:       generateDeeplyNestedArray(200),
+			input:       generateDeeplyNestedArray(DefaultMaxParsingRecursionDepth + 1),
```

Comment:
So we're also changing this from 200 to 100001. Is this part of the reason, in addition to `maxParsingRecursionDepth = 100`?

Reply:
Not part of the reason behind the CI errors. I got rid of the ability to set the global max, as I had to remove the only use for it anyways. Alternatively, I could have made this set/reset of the global limit a test-local thing, using `t.Cleanup()`, but I don't think the original optimization was necessary.